### DISCUSSION TITLE
Remote Mixer

### DISCUSSION THREAD

**Discussion by Ahh6483** [musescore/MuseScore#14614] [Ideas]
Hello there!

I just wanted to create this to bring up the idea of a remote mixer. For example one would be able to use a secondary device (such as an iPad) to control the levels and patches for Musescore. When working on a laptop, I try to save as much screen real estate as I can, and if I was able to have my mixer on a remote display like an iPad where I can physically adjust the levels would save me a lot pain of hiding and showing the mixer.

Logic Pro Remote (for iPad that works with a host workstation) has something similar embedded into the main program. A lot of digital audio mixers also have a similar app.

![2AAD7160-2C6F-47BF-8E6E-D7B543EAA2B1](https://user-images.githubusercontent.com/79336035/202893417-2a7141a5-5a11-4c9d-bf3a-8873fd821e0f.jpeg)
![1CEA4891-9EE5-4ECB-B396-0CDD2977EA93](https://user-images.githubusercontent.com/79336035/202893420-e793c03b-a33a-4c9e-bb1a-1302a415623d.jpeg)
![41558EE0-6A7F-4D13-840F-9DC4F3C31941](https://user-images.githubusercontent.com/79336035/202893421-aeae1165-15b8-46f4-925c-d67c5d64317b.jpeg)
 I've attached a few images to provide examples.

Thank you for your time and let me know if there are any questions.

**Comment by cbjeukendrup** [comment]
I'm hoping I will some day get the chance to build a full "MuseScore Remote" app for iPad... but communication between Mac and iPad isn't quite trivial. I have some experience with building an USB connection though, using a library called Peertalk. It seems to be still the only available solution.

As an temporary solution, if you have a Mac and a somewhat new iPad, I believe you can use Apple's Sidecar feature to use your iPad as a second display for your Mac. You can then drag the mixer panel out of the MuseScore window, and move it to your iPad's screen. I'm not sure if you can also use the iPad's touch screen to control the Mac in this case, but at least it lets you have the Mixer on a separate screen, so that you can use all space of your Mac's screen for the score.

**Comment by jeetee** [comment]
You used to be able to use OSC for this, but it was limited to the first 12 mixer channels unfortunately.
